jbmc/src/java_bytecode/java_bytecode_concurrency_instrumentation.cpp

```diff
@@ -361,3 +497,58 @@ void convert_threadblock(symbol_tablet &symbol_table)
     }
   }
 }
+
+/// Iterate through the symbol table to find and appropriately instrument
+/// synchronized methods.
+///
+/// A synchronized method is a method with the synchronization keyword. Any
+/// code inside of such methods has to be thread-safe. To this end, a call to
+/// "java::java.lang.Object.monitorenter:(Ljava/lang/Object;)V" and
+/// "java::java.lang.Object.monitorexit:(Ljava/lang/Object;)V" is instrumented
+/// at the start and end of the method. Note, that the former is instrumented
+/// at every statement where the execution can exit the method in question.
+/// Specifically, out of order return statements and exceptions. The latter
+/// is dealt with by instrumenting a try catch block.
+///
+/// Note: current version cannot handle static methods.
```

Comment:
Describe here what the effect of this lack of support will be for those methods.

Reply:
done